client/cmd/dexcctl/simnet-setup.sh

```diff
@@ -2,6 +2,9 @@
 # Set up DCR and BTC wallets and register with the DEX.
 # dcrdex, dexc, and the wallet simnet harnesses should all be running before
 # calling this script.
+#
+# dexc should be built with -ldflags "-X 'decred.org/dcrdex/dex.testLockTimeTaker=30s' -X 'decred.org/dcrdex/dex.testLockTimeMaker=1m'"
+# in order to mirror the default server simnet locktimes.
```

Comment:
BTW, when doing interactive testing, I usually rebuild server with longer lock times.
With eth and propagation often taking a minute on testnet, I suspect we'll want to change increase this.

Reply:
Any thoughts on using a Makefile for this stuff?

Reply:
Could.  I like your `build` and `run` scripts you added for server though.  But to start up the harness without the short lock times, I can see how a make file would be nice, like `make interactive` for longer lock times suitable for interactive testing vs `make scripted` suitable for the scripted harness trade tests.

@JoeGruffins I'm not sure what everyone's workflow looks like, but I should clarify that in general, I advise against any kind of interactive tests with these 30sec/1min lock times.  They are only good for for the *scripted* simnet tests.

Reply:
Changed the wording to only inform it can be changed in that manner.